### ISSUE TITLE
A11y: [New room] Radio buttons and controls have insufficient focus...

### ISSUE THREAD

**Issue by mxandreas** [element-hq/element-x-android#6397]
- **Checkpoint ID:** 11.1.4.11/6.1
- **Component:** New room

### Issue Description
Radio buttons and controls have insufficient focus indicator contrast.

### Recommendation
Ensure focus indicators meet 3:1 minimum contrast requirement.

--- 
*Report Date: 25.02.2026*

**Comment by bmarty** [comment]
Discussed with @gaelledel , the radio buttons / switches should also render the focused state when the list item are focused.

**Comment by bmarty** [comment]
Here is a first implementation, you can see on second item, which is focused, that the switch also renders a focused state:

<img width="357" height="159" alt="Image" src="https://github.com/user-attachments/assets/355c3ebb-7c6b-45d6-a6e0-2433f1cdaac5" />

I do not think is is better though, what do you think @gaelledel ?

**Comment by gaelledel** [comment]
No It is making things worse now because the text and BG contrast is insufficient. Bear with me whilst I investigate this.
